src/EPR.Calculator.API.Data/DataModels/CalculatorRunPomDataDetail.cs

```diff
@@ -1,23 +1,23 @@
-﻿using Microsoft.EntityFrameworkCore;
-using System.ComponentModel.DataAnnotations;
+﻿using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
 using System.Diagnostics.CodeAnalysis;
 
 namespace EPR.Calculator.API.Data.DataModels
 {
     [ExcludeFromCodeCoverage]
     [Table("calculator_run_pom_data_detail")]
-    [PrimaryKey(nameof(OrganisationId), nameof(SubsidaryId))]
     public class CalculatorRunPomDataDetail
     {
+        [Required]
+        public int Id { get; set; }
 
         [Column("organisation_id")]
         [StringLength(400)]
-        public required string OrganisationId { get; set; }
+        public string? OrganisationId { get; set; }
 
         [Column("subsidiary_id")]
         [StringLength(400)]
-        public required string SubsidaryId { get; set; }
+        public string? SubsidaryId { get; set; }
 
         [Column("submission_period")]
```

Comment:
I can't see the submission_period_desc column which is in the ADR.

Reply:
It will be part of a different ticket.

Reply:
Agreed  with @johnbenjamin-defra in a call that this will be a different ticket